frontend/src/components/organisms/Table/style.css

```diff
@@ -46,12 +46,35 @@
   transform: translate(-50%, -50%);
 }
 
+.datapass-table .header-container {
+  height: 70px;
+  display: flex;
+  flex-direction: column;
+  gap: 0.3rem;
+}
+
+.datapass-checkbox-filter.fr-checkbox-group input[type=checkbox]+label {
+  padding: 0.4rem 0;
+}
+
+.datapass-checkbox-filter.fr-checkbox-group input[type=checkbox]+label:before {
+  margin: 0.4rem 0;
+}
+
+.fr-label + .fr-input, .fr-label + .fr-input-wrap, .fr-label + .fr-select {
```

Comment:
this rule is not selective enought and can have unexpected effect on the app. Plz scope this under .datapass-table selector.